### PR TITLE
fix: addressComplete wcag legend

### LINKED ISSUE #4405: AddressComplete : WCAG Fieldset Legend

### PR DESCRIPTION
**Pull request by ShadeWyrm**
Adds missing design system css for fieldset / fieldset:legend.

Closes #4405

### CODE REVIEW

File: styles/_fieldset.scss [hunk 1]
```diff
@@ -0,0 +1,73 @@
+@layer reset, default, disabled, focus;
+:root {
```
Comment (timarney) [inline styles/_fieldset.scss:2]: Can you use 

@import "../node_modules/@cdssnc/gcds-tokens/build/web/css/components/fieldset.css";

I did that with the text input etc...

Would prefer if we can just get the vars from them vs the hardcoded.

**Review by thiessenp-cds — COMMENTED** [review]
The fieldset part looks great. 

I noticed that some aria-labelledby id's don't match the related label id. But that can be for another PR. This is happening in a number of places outside addressComplete.

I think the design system import looks good. But I'll leave that to Tim who has more experience with that :)

File: components/clientComponents/forms/AddressComplete/AddressComplete.tsx [hunk 2]
```diff
@@ -244,7 +249,7 @@ export const AddressComplete = (props: AddressCompleteProps): React.ReactElement
           </div>
         )}
         {!props.canadianOnly && (
-          <div className="mb-6">
+          <div className="mb-6 mt-4">
             <Label htmlFor={`${name}-country`} className="gc-label">
```
Comment (timarney) [inline components/clientComponents/forms/AddressComplete/AddressComplete.tsx:253]:
```suggestion
            <Label htmlFor={`${name}-country`}>
```

Comment (timarney) [reply components/clientComponents/forms/AddressComplete/AddressComplete.tsx:253]: Screenshot 2024-10-29 at 3 01 08 PM

**Review by timarney — APPROVED** [review]

**Review by timarney — APPROVED** [review]

**Pull request merged**